Makefile

```diff
@@ -2,42 +2,43 @@
 
 NOJESSIE ?= 1
 NOSTRETCH ?= 0
+export SONIC_OVERRIDE_BUILD_VARS += DBGSH=$(DBGSH)
 
 %::
 	@echo "+++ --- Making $@ --- +++"
 ifeq ($(NOJESSIE), 0)
-	EXTRA_DOCKER_TARGETS=$(notdir $@) make -f Makefile.work jessie
+	EXTRA_DOCKER_TARGETS=$(notdir $@) $(MAKE) -f Makefile.work jessie
```

Comment:
where is this MAKE defined?

Reply:
MAKE is a in-built Makefile variable.  If we use '$(MAKE )' instead of 'make', it propagate all the ENV variables to the sub make process.